Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN apt-get -q update \
     libpng12-0 \
     libx11-6 \
     libxpm4 \
-    mysql-client
+    mysql-client \
+    postgresql-client libpq-dev
```

Comment:
please put the second line in an extra line and keep the list sorted by alphabet